src/ShopBundle/objects/BackendModules/CMSShopArticleIndex/views/standard.view.php

```diff
@@ -1,30 +1,34 @@
-<div class="CMSInterfacePopup">
-    <form name="indexer" method="post" action="<?=PATH_CMS_CONTROLLER; ?>" accept-charset="UTF-8">
-        <input type="hidden" name="pagedef" value="<?=TGlobal::OutHTML($data['pagedef']); ?>"/>
-        <input type="hidden" name="_pagedefType" value="<?= $data['_pagedefType']; ?>"/>
-        <input type="hidden" name="module_fnc[<?=TGlobal::OutHTML($data['sModuleSpotName']); ?>]"
-               value="TickerIndexGeneration"/>
+<div class="card CMSInterfacePopup">
+    <div class="card-header">
+        Article search index
+    </div>
+    <div class="card-body">
+        <form name="indexer" method="post" action="<?=PATH_CMS_CONTROLLER; ?>" accept-charset="UTF-8">
+            <input type="hidden" name="pagedef" value="<?=TGlobal::OutHTML($data['pagedef']); ?>"/>
+            <input type="hidden" name="_pagedefType" value="<?= $data['_pagedefType']; ?>"/>
+            <input type="hidden" name="module_fnc[<?=TGlobal::OutHTML($data['sModuleSpotName']); ?>]"
+                   value="TickerIndexGeneration"/>
 
-
-        <?php if (!$data['bIndexIsRunning'] || $data['bIndexCompleted']) {
-    ?>
-        <input type="submit" value="generate"/>
+            <?php if (!$data['bIndexIsRunning'] || $data['bIndexCompleted']) {
+                ?>
+                <input class="btn btn-primary" type="submit" value="generate"/>
```

Comment:
Could you also add this text to translations (and with a capital "G")?

Reply:
Done.